basic_husky_franka_controller:
    type: kimm_action_manager/BasicHuskyFrankaController
    arm_id: ns1_panda
    joint_names:
        - ns1_panda_joint1
        - ns1_panda_joint2
        - ns1_panda_joint3
        - ns1_panda_joint4
        - ns1_panda_joint5
        - ns1_panda_joint6
        - ns1_panda_joint7
    # Configure the initial defaults for the collision behavior reflexes.
    collision_config:
        lower_torque_thresholds_acceleration: [40.0, 40.0, 40.0, 40.0, 30.0, 30.0, 30.0]  # [Nm]
        upper_torque_thresholds_acceleration: [40.0, 40.0, 40.0, 30.0, 30.0, 30.0, 30.0]  # [Nm]
        lower_torque_thresholds_nominal: [40.0, 40.0, 40.0, 40.0, 30.0, 30.0, 30.0]  # [Nm]
        upper_torque_thresholds_nominal: [40.0, 40.0, 40.0, 40.0, 30.0, 30.0, 30.0]  # [Nm] 
        lower_force_thresholds_acceleration: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0]  # [N, N, N, Nm, Nm, Nm]
        upper_force_thresholds_acceleration: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0]  # [N, N, N, Nm, Nm, Nm]
        lower_force_thresholds_nominal: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0]  # [N, N, N, Nm, Nm, Nm]
        upper_force_thresholds_nominal: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0]  # [N, N, N, Nm, Nm, Nm]
